demo/components/demo/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,32 @@
+<span class="slds-dropdown-trigger" nglDropdown [(open)]="open" (select)="onSelect($event)">
+  <button class="slds-button slds-button--neutral" nglDropdownTrigger>
+    <span class="slds-truncate">Select an Action</span>
+    <svg aria-hidden="true" class="slds-button__icon slds-button__icon--hint">
+      <use xlink:href="assets/icons/utility-sprite/svg/symbols.svg#down"></use>
+    </svg>
+  </button>
+  <div class="slds-dropdown slds-dropdown--left">
+    <ul class="dropdown__list" role="menu">
+      <li class="slds-dropdown__item" [nglDropdownItem]="'Action1'">
+        <a role="menuitem">
+          <p class="slds-truncate">Action1</p>
+          <svg aria-hidden="true" class="slds-icon slds-icon--x-small slds-icon-text-default slds-m-left--small slds-shrink-none">
```

Comment:
Yes, this is there on purpose and it's not related to the `aria-selected` attribute placed on the parent `li`. Please, refer to https://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/ and check the CSS at https://www.lightningdesignsystem.com/components/menus/, specifically the demo CSS of the section titled `Single Icon - Left`.
In short, the `aria-hidden` on the svg there, is placed by the slds system, as I understand it, to reduce the impact of the check mark (it's just an icon) when using assistive tools. An assistive tool would read the selected `li` but the check mark is of no significance. They also use it in their demo page even thought the item has been selected.
Have I understood something wrong?